app/views/public_activity/approval/_update.html.erb

```diff
@@ -0,0 +1,7 @@
+Approved an <%= link_to_activity(activity) %>
+<% if activity.trackable %>
+<br/>
+  Approvable name: <%= activity.trackable.approvable.email %>
+<% else %>
+which has since been revoked.
```

Comment:
There is an indentation within `else` case in `_create` partial. And there is no here.